manifests/params.pp

```diff
@@ -202,6 +202,7 @@
 
   # puppetca settings
   $puppetca              = true
+  $puppetca_split_configs = true
```

Comment:
My eyes! My eyes! If only the parameter was `puppetca_split_config` (not a serious suggestion) the indentation would be alligned :)